doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-global-settings-panel-dmn-based-proc.adoc

```diff
@@ -0,0 +1,15 @@
+[id='test-designer-global-settings-panel-dmn-based-proc']
+= Global settings for DMN-based test scenarios
+
+Follow the procedure below to view and edit the global settings of the DMN-based test scenarios.
+
+.Procedure
+. Click `Settings` tab on the right side of the test scenario designer to display the attributes.
+. Configure the following attributes in the `Settings` panel.
```

Comment:
Use colons at the end of the step if there are sub-steps or lists, see IBMSG "Lists" p. 75 and "Colons after introductory text" p. 43.